database/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
 
 # Create your models here.
+class Request(models.Model):
+    request_ID = models.IntegerField(max_length=50)
+    user_access_ID= models.CharField(max_length=250)
+    request_time = models.DateTimeField(max_length=100)
+
+class User(models.Model):
+    user_id = models.IntegerField(max_length=50)
+    email = models.EmailField(max_length=100)
+    password = models.CharField(max_length=250)
+    user_type = models.CharField(max_length=20)
+
+class Adviser(models.Model):
+    adviser_id = models.IntegerField(max_length=50)
+    full_name = models.CharField(max_length=250)
+    email_address = models.EmailField(max_length=100)
+    password = models.CharField(max_length=250)
+
+class Section(models.Model):
+    section_id = models.IntegerField(max_length=50)
+    year_level = models.CharField(max_length=10)
+    name = models.CharField(max_length=20)
+
+class Schoolyear(models.Model):
```

Comment:
Just convention: make models have title case (SchoolYear and not Schoolyear)

Reply:
done